<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ticks Player</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QVBoxLayout" name="vertical_layout_3">
            <item>
             <widget class="QLabel" name="mode_label">
              <property name="text">
               <string>Mode</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="mode_combobox"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="mode_go_push_button">
            <property name="text">
             <string>Go</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="vertical_spacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="vertical_layout_4">
            <item>
             <widget class="QLabel" name="view_label">
              <property name="text">
               <string>View</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="view_combobox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="indicators_group">
          <property name="title">
           <string>Indicators</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="vertical_layout_7">
             <item>
              <layout class="QVBoxLayout" name="vertical_layout">
               <item>
                <widget class="QLabel" name="on_graph_label">
                 <property name="text">
                  <string>On Graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="on_graph_combobox"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="vertical_layout_2">
               <item>
                <widget class="QLabel" name="off_graph_label">
                 <property name="text">
                  <string>Off Graph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="off_graph_combobox"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
          <zorder></zorder>
          <zorder></zorder>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="start_refresh_button">
          <property name="text">
           <string>Start/Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="chart_group">
        <property name="title">
         <string>Chart</string>
        </property>
        <zorder></zorder>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
